nemo/collections/common/tokenizers/text_to_speech/tokenizer_wrapper.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nemo.collections.common.tokenizers import TokenizerSpec
+from nemo.collections.common.tokenizers.text_to_speech.g2ps import EnglishG2p
+from nemo.collections.common.tokenizers.text_to_speech.tts_tokenizers import EnglishPhonemesTokenizer
+
+__all__ = ['TextToSpeechTokenizer']
+
+
+class TextToSpeechTokenizer(TokenizerSpec):
```

Comment:
I can confirm that older checkpoints work but it was done by importing all tts_tokenizer related classes from `collections.common.tokenizers.text_to_speech` in the `collections.tts.torch` files (see `collections/tts/torch/tts_tokenizers.py` and `collections/tts/torch/g2ps.py` for the ugly workaround we would remove in the future). I did not change anything in current tts tokenizers and just moved them to other location.

This particular wrapper is currently only used by myself in my ST code (not pushed yet), so I can tokenize text in NMT dataset and pass it to TTS.

After all files are moved, I will work on another PR which refactors current TTS tokenizers around TokenizerSpec (basically merging them with current wrapper). This will require more time and testing as the backward compatibility will be definitely broken. We will not need to re-train our released checkpoints, though. We just need to replace the config files inside `.nemo` tar files.

Reply:
Sounds good, thanks! That should be relatively straightforward then, thankfully.